test/src/test/java/GroupByTest.java

```diff
@@ -61,7 +61,7 @@ public void testGroupByOrderByCount() {
 
     @Test
     public void testGroupByWithFilter() {
-        getRequest(request2path("groupByWithFilter.json"))
+        getRequest(request2path("groupByWIthFilter.json"))
```

Comment:
Is this what you intended?
Do you care?

Reply:
That's why the test was failing, and I've seen things get weird when changing the case of a character in a filename - figured the file will go away eventually in favor of building an object instead.